src/ScrollList/index.tsx

```diff
@@ -74,31 +74,48 @@ FixedSizeList.displayName = 'FixedSizeList';
 /**
  * VARIABLE SIZE LIST
  */
-// interface IVariableSizeListProps extends Omit<ReactWindow.VariableSizeListProps, 'height' | 'width'> {
-//   className?: string;
-//   height: number | string;
-//   width: number | string;
-//   shadows?: boolean;
-// }
+interface IVariableSizeListProps extends Omit<ReactWindow.VariableSizeListProps, 'height' | 'width'> {
+  className?: string;
+}
 
-// const VariableSizeList: React.FunctionComponent<IVariableSizeListProps> = React.forwardRef<
-//   ReactWindow.VariableSizeList,
-//   IVariableSizeListProps
-// >(function VariableSizeList(props, ref) {
-//   const { className, children, ...rest } = props;
+const VariableSizeList: React.FunctionComponent<IVariableSizeListProps> = React.forwardRef<
+  ReactWindow.VariableSizeList,
+  IVariableSizeListProps
+>(function VariableSizeList(props, ref) {
+  const { children, ...rest } = props;
 
-//   return (
-//     <ReactWindow.VariableSizeList {...rest} outerRef={ref} outerElementType={ScrollContainer}>
-//       {children}
-//     </ReactWindow.VariableSizeList>
-//   );
-// });
+  return (
+    <div className="ScrollList-Container h-full">
+      <AutoSizer>
+        {({ height, width }) => (
+          <ReactWindow.VariableSizeList
+            {...rest}
+            height={height}
+            width={width}
+            // className gets passed to ScrollList-Content
```

Comment:
These comments are a bit weird here since you're not destructing those props

Reply:
You are right. Forgot to remove them.